omgeo/services/__init__.py

```diff
@@ -658,9 +658,11 @@ def _geocode(self, pq):
                 query = dict(query, searchExtent=pq.viewbox.to_esri_wgs_json())
         else: # single-line
             method = 'find'
+            magic_key = pq.key if hasattr(pq, 'key') else ''
```

Comment:
I think we should add this PlaceQuery option to the documentation. I would recommend checking with the Cicero team to see where they would like to put it. It is ESRI specific, so it should probably not be under the generic PlaceQuery description.